code/game/machinery/_machinery.dm

```diff
@@ -132,6 +132,9 @@ Class Procs:
 	var/tgui_id // ID of TGUI interface
 	var/ui_style // ID of custom TGUI style (optional)
 
+	//Maximum time an EMP will disable this machine for
+	var/emp_susceptibility = 2 MINUTES
```

Comment:
this var is extremely unclear (at least to me)

Reply:
its the Maximum time an EMP will disable this machine for

Reply:
Why not name this maximum_emp_time 🤔